app/controllers/merchant_items_controller.rb

```diff
@@ -0,0 +1,13 @@
+class MerchantItemsController < ApplicationController
+
+def show
+  @merchant = Merchant.find(params[:merchant_id])
+  @item = @merchant.items.find(params[:item_id])
```

Comment:
Should we move lines 5 and 10 to a method?

Reply:
I think thats a good idea to make those lines into a model method.